src/openrct2/GameState.h

```diff
@@ -37,121 +37,121 @@ namespace OpenRCT2
 {
     struct GameState_t
     {
-        ::OpenRCT2::Park::ParkData Park{};
-        std::string PluginStorage;
-        uint32_t CurrentTicks{};
-        ::OpenRCT2::Date Date;
-        money64 CompanyValue;
+        ::OpenRCT2::Park::ParkData park{};
+        std::string pluginStorage;
+        uint32_t currentTicks{};
+        ::OpenRCT2::Date date;
```

Comment:
Are these namespace resolutions still needed now that their names no longer clash with the properties? i.e. does this now work?
```suggestion
        Park::ParkData park{};
        std::string pluginStorage;
        uint32_t currentTicks{};
        Date date;
```